setup.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pathlib
+
+from setuptools import setup, find_packages
+
+# The directory containing this file
+HERE = pathlib.Path(__file__).parent
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+# This call to setup() does all the work
+setup(
+    name="backend_workshop_microservices_common",
```

Comment:
It is not a stopper but I think that the name is like too long. Dunno if it would be better to try something shorter like quservices_library

Reply:
sure! I will upload the new repo to pypi now